docs/source/customizations/native.mdx

```diff
@@ -1,83 +1,33 @@
 ---
 title: Writing native Rust plugins for the Apollo Router
-description: Building and compiling a custom plugin for the Apollo Router with Rust
+description: Extend the Apollo Router with custom Rust code
 ---
 
 import { Link } from 'gatsby';
 
-Your organisation may have specific needs that cannot be expressed as generic functionality, and will therefore not be available via core Apollo Router functionality.
-
-For instance:
-* Bespoke authentication/authorization
-* Bespoke logging
-* Bespoke tracing
-
-The plugin mechanism allows you to add such functionality to the Apollo Router.
-
-## Router pipeline
```

Comment:
I moved this stuff out into the plugin overview because it felt relevant to the other plugin type as well